requirements/requirements_doc.txt

```diff
@@ -1,7 +1,6 @@
 matplotlib==3.7.1
-ansys-fluent-core==0.16.0
-pyvista==0.38.5
-ansys-mapdl-core==0.64.0
+ansys-fluent-core>=0.16.0
+ansys-mapdl-core>=0.65.0,<0.66.0
```

Comment:
Could you please recall me why we limited ourselves to 0.65.0? I remember it was related to a mapdl issue but wasn't-it this issue resolved now?

Reply:
related to [Issue2502](https://github.com/ansys/pymapdl/issues/2502) and said to be fixed in [PR2537](https://github.com/ansys/pymapdl/pull/2537)